xc/xenops_server_xen.ml

```diff
@@ -1069,7 +1069,7 @@ module VM = struct
 		Domain.set_xsdata ~xs di.Xenctrl.domid xsdata
 	) Newest task vm
 
-	let set_vcpus task vm target = on_domain (fun xc xs _ _ di ->
+	let set_vcpus_trad task vm target = on_domain (fun xc xs _ _ di ->
```

Comment:
As Ross says, this isn't just for trad, so the name should reflect that.

Reply:
Thanks, updated